packages/sdk/test/unit/GroupKeyManager.test.ts

```diff
@@ -29,6 +30,7 @@ describe('GroupKeyManager', () => {
         return new GroupKeyManager(
             subscriberKeyExchange,
             groupKeyStore,
+            new StreamIDBuilder(EthereumKeyPairIdentity.fromPrivateKey(wallet.privateKey)),
```

Comment:
Let's use local variable `const identity = ...` so that we can use it at lines 33 and 40